@xen-orchestra/lite/src/stories/web-core/card/card-title.story.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      prop('count').num().widget(),
+      prop('counterColor').str().enum('info', 'error', 'success', 'warning', 'black').widget(),
+      slot(),
+      setting('defaultSlotContent').preset('Legend title').widget(text()),
+      setting('rightSlotContent').preset('Info').widget(text()),
+      setting('bottomLeftSlotContent').preset('Description').widget(text()),
+    ]"
+  >
+    <CardTitle v-bind="properties"
+      ><template #left>{{ settings.defaultSlotContent }}</template>
+      <template #right>{{ settings.rightSlotContent }}</template>
+      <template #bottom-left>{{ settings.bottomLeftSlotContent }}</template>
+      <UiButton level="tertiary">Manage<UiIcon :icon="faAngleRight" color="info" /></UiButton>
```

Comment:
The `UiButton` component already has two props for managing icons (`:left-icon` and `:right-icon`):
```suggestion
      <UiButton level="tertiary" :right-icon="faAngleRight">Manage</UiButton>
```